release_docs

```diff
@@ -109,6 +109,7 @@ files_to_copy=(
     "source/recipes/plot_5_recipe.py"
     "source/recipes/plot_6_recipe.ipynb"
     "source/recipes/plot_6_recipe.py"
+    "source/recipes/plot_7_recipe.py"
```

Comment:
Sorry to chime in randomly here, but (I was just having a look at how the Sphinx gallery you've set up works, out of curiosity) and I'm thinking it might help us keep track of the various recipe files if you create a separate list of the root file names, e.g:

```python
recipes_files = (
    "recipes/plot_1_recipe",
    ...
    "recipes/plot_7_recipe",
)
```

and then you can add them with their various extensions to the list above and below (for the similarly-named HTML files) via construction from those (and e.g. list comprehensions), e.g. `files_to_copy = [<others>] + [f"source/{name}.py" for name in recipes_files] + ...`.

That way you'd only have to update one with each new recipe. Just an idea if you have time to do a little code refactoring!

Reply:
I have done the changes in: 41c88e623873c1f29c54aacdc0230319320288d4, but I was thinking about making it more concise so that one doesn't have to keep changing the `release_docs` with every added recipe. Something like:
```console
recipes_dir="source/recipes"
recipes_files=( "${recipes_dir}"/*_{recipe,helper}.{ipynb,py} )

files_to_copy=(
    "${recipes_dir}/recipes_jupyter.zip"
    "${recipes_dir}/recipes_python.zip"
    "${recipes_files[@]}"
)

destination_dir="$dir/_downloads"

cp -pv "${files_to_copy[@]}" "$destination_dir"

html_files=(
    "$dir/recipes/index.html"
    "$dir/recipes/"*.html
)

for html_file in "${html_files[@]}"; do
    for recipe_file in "${recipes_files[@]}"; do
        sed -i "s|href=\".*/$(basename "$recipe_file")\"|href=\"../_downloads/$(basename "$recipe_file")\"|g" "$html_file"
    done
done
```
What do you think?

Reply:
Looks good